cognite_toolkit/_cdf_tk/templates/_utils.py

```diff
@@ -108,3 +109,13 @@ def _get_cognite_module_version(project_dir: Path) -> str:
             "'_system.yaml' or 'environments.yaml' (before 0.1.0b6) file?"
         )
     return previous_version
+
+
+def _search_system_yaml(project_dir: Path) -> Path | None:
+    if (project_dir / "_system.yaml").exists():
+        return project_dir / "_system.yaml"
+    if (project_dir / COGNITE_MODULES / "_system.yaml").exists():
+        return project_dir / COGNITE_MODULES / "_system.yaml"
```

Comment:
Are these even needed if we can `rglob` anyway?

Reply:
Yes, it ensures that we check there first.

Reply:
Added a comment.